tools/train_linemod_template.cpp

```diff
@@ -299,4 +306,12 @@ main (int argc, char** argv)
     print_error("All input pcd files are unorganized.\n");
   }
 
+  if (error_detected)
+  {
+    return -1;
+  }
+  else
+  {
+    return 0;
+  }
```

Comment:
Return codes from applications are usually non-negative integers. In this situation I would literally just do
```cpp
return error_detected;
```
I leave it you the decision of the (non)explicit cast.

Reply:
PTAL

Reply:
:+1:  It's still returning a negative integer though. Any reason for that choice?

Reply:
It was returning a [negative integer](https://github.com/PointCloudLibrary/pcl/pull/3652/files#diff-66720143805e877dec9fbeaa90a0326cL270) before. I kept the same behavior. No other rationale.